incubator/virtualcluster/config/setup/all_in_one.yaml

```diff
@@ -215,13 +215,70 @@ spec:
         name: vc-manager
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: vc-syncer-role
+rules:
+- apiGroups:
+    - ""
+  resources:
+    - configmaps
+    - endpoints
+    - namespaces
+    - pods
+    - secrets
+    - services
+    - serviceaccounts
+    - persistentvolumeclaims
+  verbs:
+    - get
+    - list
+    - watch
+    - create
+    - update
+    - patch
+    - delete
+- apiGroups:
+    - ""
+  resources:
+    - events
+    - nodes
+    - persistentvolumes
+    - storageclasses
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - ""
+  resources:
+    - namespaces/status
+    - pods/status
+    - services/status
+    - nodes/status
+    - persistentvolumes/status
+    - persistentvolumeclaims/status
+  verbs:
+    - get
+    - update
```

Comment:
for all the status in super master, syncer should not have update/patch permission.